_sass/color_schemes/featurebase.scss

```diff
@@ -17,3 +17,11 @@ $link-color: $purple-000;
 
 // Code
 $code-background-color: #F7F7FB;
+
+// Syntax highlighting with dark purple text
+// makes code block legible in table headings
+
+.language-plaintext, .highlighter-rouge {
```

Comment:
This is the fix so code blocks are legible in table headers.

Reply:
Up to you to make this but PLEASE do validation that this change doesn't mess up other formatting elsewhere. I've seen that before, and I'd rather just not have codeblock in table headers than break other formatting as neither of us know css well. Change up to you but validation throughout docs (visual inspection) is needed